runhouse/main.py

```diff
@@ -679,6 +303,207 @@ def cluster_list(
         print_sky_clusters_msg(len(sky_clusters))
 
 
+@cluster_app.command("keep-warm")
+def cluster_keep_warm(
+    cluster_name: str = typer.Argument(
+        None,
+        help="Name of cluster to keep warm. If not specified will check the local cluster.",
+    ),
+    mins: Optional[int] = typer.Option(
+        -1,
+        help="Amount of time (in min) to keep the cluster warm after inactivity. If set to -1, keeps cluster warm indefinitely.",
+    ),
+):
+    """Keep the cluster warm for given number of minutes after inactivity.
+
+    Example:
+
+        ``$ runhouse cluster keep-warm /sashab/rh-basic-cpu``
+
+    """
+
+    current_cluster = get_cluster_or_local(cluster_name)
+
+    try:
+        if not current_cluster.is_up():
+            print_bring_cluster_up_msg(cluster_name=cluster_name)
+            return
+        current_cluster.keep_warm(mins=mins)
+    except ValueError:
+        console.print(f"{cluster_name} is not saved in Den.")
+        sky_live_clusters = get_all_sky_clusters()
```

Comment:
thoughts on providing a `refresh` param for the commands that call `sky.status()`?

Reply:
hmmm, not sure if I'm in favor to add the refresh param...

I think we had some discussion about it and we decided that currently if there are live sky clusters, we'll print the following msg to the console: `There are X live sky clusters that are not saved in Den. For more information, please run [bold italic]sky status -r[/bold italic].` This way, we make the user explicitly run the sky status command with the refresh parameter, which strengthens the value of saving clusters to Den (since we're getting the live status "out of the box" without needing the refresh parameter).

Our main goal is to encourage users to rely on Den as much as possible. By providing the refresh parameter, we introduce a convenient and easy-to-use wrapper around sky CLI commands. This makes it more straightforward to use sky via Runhouse rather than using Den without saving clusters to it.

wdyt?